tester/paddle_to_torch/rules.py

```diff
@@ -1789,6 +1832,19 @@ def to_float_if_needed(tensor):
         )
         return ConvertResult.success(paddle_api, code)
 
+class LogNormalRule(BaseRule):
+    def apply(self, paddle_api: str) -> ConvertResult:
+        defaults_code, map_code = self.apply_generic()
+        pre = """
+"""
+        core = f"result = {self.torch_api}(**_kwargs)"
+        post = "result = torch.exp(result)"
+        code = Code(
+            preprocess=defaults_code + pre.splitlines() + map_code,
+            core=core.splitlines(),
+            postprocess=post.splitlines()
+        )
+        return ConvertResult.success(paddle_api, code)
```

Comment:
这样应该不算对应？此外，core与post均为执行逻辑，可修改为：
```suggestion
class LogNormalRule(BaseRule):
    def apply(self, paddle_api: str) -> ConvertResult:
        defaults_code, map_code = self.apply_generic()
        core = """
result = torch.normal(**_kwargs)"
result = torch.exp(result)
"""
        code = Code(
            preprocess=defaults_code +map_code,
            core=core.splitlines(),
        )
        return ConvertResult.success(paddle_api, code)
```